docs/advanced_examples/LoraMLP.ipynb

```diff
@@ -21,7 +21,7 @@
     {
```

Comment:
I suggest: The fine-tuning dataset and the trained LoRA weights are protected using encryption. Thus, training can be securely outsourced to a remote server without compromising any sensitive data.

Reply:
... The forward and backward passes on these layers are performed using encrypted activations and gradients. Meanwhile, the LoRA weights are kept by the client, which performs locally the forward and backward passes on the LoRA weights.